src/Pages/2DPrinting/App.js

```diff
@@ -16,180 +11,149 @@ import {
-              <Input type='select' name='select' id='exampleSelect'>
-                <option>1</option>
-                <option>2</option>
-                <option>3</option>
-                <option>4</option>
-                <option>5</option>
-              </Input>
-              <legend>Type of Print</legend>
-              <FormGroup check>
-                <Label check>
-                  <Input type='radio' name='radio1' /> Front
-                </Label>
-              </FormGroup>
-              <FormGroup check>
-                <Label check>
-                  <Input type='radio' name='radio1' />{' '}
-                  {/* add disabled if in need of option disablement */}
-                  Front & Back
-                </Label>
-              </FormGroup>
+export default function Printing (props) {
+  const [modal, setModal] = useState(false)
+  const [nestedModal, setNestedModal] = useState(false)
+  const [closeAll, setCloseAll] = useState(false)
+  const [files, setFiles] = useState([])
+  const toggle = false
+
+  return (
+    <div>
+      <Jumbotron>
+        <div className='text-center'>
+          <h1 className='display-4'>SCE Printing System</h1>{' '}
+        </div>
+      </Jumbotron>
+
+      <div className='printInfo'>
+        <p>
+          Welcome to printing! Click the icon below and upload your file. Each
+          member can print up to 30 pages a week.
+        </p>
+      </div>
+
+      {props.buttonLabel}
+      <FilePond
+        files={files}
+        allowMultiple={false}
+        acceptedFileTypes='application/pdf'
+        maxFileSize='10MB'
+        // server={this.serverOptions}
+        // oninit={() => this.handleInit()}
+        onupdatefiles={fileItems => {
+          // Set currently active file objects to this.state
+          setFiles(fileItems.map(fileItem => fileItem.file))
+        }}
+        labelIdle="Drag & Drop or Touch Here <br />
+
+            <svg aria-hidden='true' viewBox='0 0 512 512' width='40%'>
+              <path
+                d='M399.95 160h-287.9C76.824 160 48 188.803 48
+              224v138.667h79.899V448H384.1v-85.333H464V224c0-35.197-28.825-64-64.05-64zM352
+              416H160V288h192v128zm32.101-352H127.899v80H384.1V64z'
+              />
+            </svg>"
+      />
+      <Button
+        color='primary'
+        className='continue'
+        onClick={() => setModal(!modal)}
+      >
+        continue
+      </Button>
+      <Modal
+        isOpen={modal}
+        toggle={() => setModal(!modal)}
+        className={props.className}
+        onClosed={closeAll ? toggle : undefined}
+      >
+        <ModalHeader toggle={() => setModal(!modal)}>
+          Software & Computer Engineering Society
+        </ModalHeader>
+        <ModalBody>
+          <br />
+          <FormGroup>
+            <legend for='exampleSelect'>Number of copies</legend>
+            <Input type='select' name='select' id='exampleSelect'>
```

Comment:
this should instead be a numeric input, with numbers ranging from 1 to the limit of allowed copies for the user. For this PR, please change this to be an input from 1-30.

[this guide](https://www.w3schools.com/tags/att_input_type_number.asp) is helpful for numeric inputs, e.g.
```html
<Input type="number" name="points" min="1" max="30" step="1">
```